src/assets/locales/de.json

```
@@ -55,7 +55,7 @@
         "text": "Hast du es korrekt konfiguriert?",
         "title": "Server nicht erreichbar"
       },
-      "host": "Du verbindest dich zu <0>{{hostname}}</0> - stelle sicher, dass du diesem vertraust, bevor du einen Account erstellst",
+      "host": "Du verbindest dich mit <0>{{hostname}}</0> - stelle sicher das du dem Server vertraust bevor du ein Konto erstellst",
```

This improvement didn't improve everything, it should be like this:

```suggestion
      "host": "Du verbindest dich mit <0>{{hostname}}</0> - stelle sicher, dass du dem Server vertraust, bevor du ein Konto erstellst",
```

Is there a Crowdin page for translations or should I just edit the source file manually?